scss/_nav.scss

```diff
@@ -135,6 +135,10 @@
       outline-color: currentcolor;
     }
 
+    &.active:hover {
+      color: var(--#{$prefix}nav-tabs-link-active-color);
+    }
+
```

Comment:
OK with this.

 ⚠️ There is a remaining visual behavior to be fixed in PR #1437: on mouse down (active state) on a selected item, the text color stays black as expected, but when the click is released and the mouse is moved away, 
the text color turns orange when it shouldn't. It has to do with the focus management.